content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,117 @@
+## "Send To" feature and Shell improvements
+One of the main highlights of this release is the amount of improvements done to the Shell component, which makes up a vital part of the ReactOS user experience.
+Katayama Hirofumi MZ is the pioneer of the "Send To" implementation, a feature of the Shell that can be used to send files or directories to a certain location.
+
+In addition, "Open File Location" and "Open Command prompt here" were also implemented thanks to him.
+Apart from the aforementioned features, let's not forget the bug fixes that affected the operability of the Shell.
+Katayama Hirofumi MZ fixed the scroll selection and Mark Jansen improved some COM (Component Object Model) interfaces, so that certain drag and drop regressions could be fixed.
+The implementations of "Send To" and "Open File Location" are shown in the screenshots below:
+
+{{< figure link="/img/project-news/reactos-0414-released/shell.png" src="/img/project-news/reactos-0414-released/shell.png" >}}
+
+## NEC PC-9800 boot support
+NEC PC-9800 (or PC-98 in short) is a series of 16-bit and 32-bit Japanese computers developed and manufactured by NEC.
+As that type of hardware is based on x86 processors, it is a relatively easy porting target.
+Nevertheless, every port to another architecture opens up possibilities for finding and fixing bugs in the kernel modules of ReactOS.
+
+Dmitry Borisov, a new ReactOS contributor, helped to get the PC-98 port in shape.
+While there is stuff left to be done in the development of PC-98 boot support, Dmitry Borisov made great strides to get ReactOS booting on that platform.
+The following screenshot below demonstrates ReactOS booting on Neko Project 21/W emulator.
+
+{{< figure link="/img/project-news/reactos-0414-released/nec98.png" src="/img/project-news/reactos-0414-released/nec98.png" >}}
+
+## ICMP improvements
+Internet Control Message Protocol (or ICMP by short) is a protocol used by network device drivers to send network information such as errors or log status.
+The implementation was rather scarce and incomplete with pieces of code unimplemented or unfinished.
+The IP Helper module primarily uses this protocol and as a matter of that, it brought a lot of issues and bugs. 
+
+Thanks to Victor Perevertkin, and with the help of Tim Crawford, the ICMP protocol implementation has seen lots of improvements.
+These range from the implementation of the IOCTL_ICMP_ECHO_REQUEST I/O control code to a full  rewrite of the Icmp** routines.
+As a result, the network device drivers can now handle ICMP requests in a correct manner.
+This is proven in the screenshot below, which shows the tracert command-line utility now outputting complete information.
+
+{{< figure link="/img/project-news/reactos-0414-released/icmp.png" src="/img/project-news/reactos-0414-released/icmp.png" >}}
+
+## Kernel improvements
+The NT kernel of ReactOS is the heart that is responsible for the well functioning of the system and communication with the bare metal.
```

Comment:
Suggestion:
The NU kernel of ReactOS is the heart that is responsible for the correct functioning..."